app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require 'prawn'
-require 'prawn/table'
```

Comment:
I didn't notice uses of Prawn in this class. I'm assuming Rspecs will fail if this removal is bad.

Reply:
Oh yes, I think we missed that during the refactor to pull out our pdf generator into another class. Good catch!

Reply:
I do wonder if we need to add this explicit require to `hypertension_pdf_generator.rb` - I imagine it would have run into test failures in its own spec if it was needed, though. 🤔

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/8558#discussion_r793896226